packages/node-core/src/configure/NodeConfig.ts

```diff
@@ -63,6 +63,7 @@ const DEFAULT_CONFIG = {
   disableHistorical: false,
   multiChain: false,
   unfinalizedBlocks: false,
+  enableSSL: false,
```

Comment:
Does this conflict with https://github.com/subquery/subql/pull/1513

Reply:
This actually not been used, remove it now.

I checked the PR, looks like not conflicting, now we are enable SSL connection by default, if DB not support his feature will try connect again without SSL. I think another PR can make change base on this one.